### PR TITLE
take away the words 'at the fork'

### LINKED ISSUE #202: Remove "at the fork" from forks
## Summary

Current behavior: "Keep right at the fork towards Foster Road."
Desired behavior: "Keep right towards Foster Road."

Right now, the words "at the fork" appear in every `fork` instruction. I propose that we remove "at the fork" from these instructions. Why?
  - it doesn't add additional context that you can't get from looking at the road
  - it makes voice instructions longer

All "fork" instructions can be seen here: https://github.com/Project-OSRM/osrm-text-instructions/blob/master/languages/translations/en.json#L157-L188

## Acceptance criteria / implementation plan

Remove "at the fork" from every `fork` instruction.

@allierowan @hdaymon @mcwhittemore

### PR DESCRIPTION
**Pull request by allierowan**
# Issue

closes #202 

## Tasklist

 - [X] remove the works "at the fork" from the en templates
 - [X] change fixtures and test
 - [x] Add changelog entry
 - [ ] Review

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

File: languages/translations/en.json [hunk 1]
```diff
@@ -157,28 +157,28 @@
...
-                "name": "Keep right at the fork onto {way_name}",
-                "destination": "Keep right at the fork towards {destination}"
+                "name": "Keep right onto {way_name}",
+                "destination": "Keep right towards {destination}"
             },
             "sharp left": {
                 "default": "Take a sharp left at the fork",
-                "name": "Take a sharp left at the fork onto {way_name}",
-                "destination": "Take a sharp left at the fork towards {destination}"
+                "name": "Take a sharp fork onto {way_name}",
```
Comment (lyzidiamond) [inline languages/translations/en.json]: Take a sharp fork --> Take a sharp left

File: languages/translations/en.json [hunk 2]
```diff
@@ -157,28 +157,28 @@
...
+                "destination": "Take a sharp fork towards {destination}"
```
Comment (lyzidiamond) [inline languages/translations/en.json]: Take a sharp fork --> Take a sharp left

File: test/fixtures/v5/fork/sharp_left_destination.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
     },
     "instructions": {
         "de": "Scharf links abbiegen an der Gabelung Richtung Destination 1",
-        "en": "Take a sharp left at the fork towards Destination 1",
+        "en": "Take a sharp fork towards Destination 1",
```
Comment (lyzidiamond) [inline test/fixtures/v5/fork/sharp_left_destination.json]: sharp fork --> sharp left

File: test/fixtures/v5/fork/sharp_left_exit.json [hunk 4]
```diff
@@ -10,7 +10,7 @@
     },
     "instructions": {
         "de": "Scharf links abbiegen an der Gabelung auf Way Name",
-        "en": "Take a sharp left at the fork onto Way Name",
+        "en": "Take a sharp fork onto Way Name",
```
Comment (lyzidiamond) [inline test/fixtures/v5/fork/sharp_left_exit.json]: sharp fork --> sharp left

File: test/fixtures/v5/fork/sharp_left_exit_destination.json [hunk 5]
```diff
@@ -11,7 +11,7 @@
     },
     "instructions": {
         "de": "Scharf links abbiegen an der Gabelung Richtung Destination 1",
-        "en": "Take a sharp left at the fork towards Destination 1",
+        "en": "Take a sharp fork towards Destination 1",
```
Comment (lyzidiamond) [inline test/fixtures/v5/fork/sharp_left_exit_destination.json]: sharp fork --> sharp left

File: test/fixtures/v5/fork/sharp_left_name.json [hunk 6]
```diff
@@ -9,7 +9,7 @@
     },
     "instructions": {
         "de": "Scharf links abbiegen an der Gabelung auf Way Name",
-        "en": "Take a sharp left at the fork onto Way Name",
+        "en": "Take a sharp fork onto Way Name",
```
Comment (lyzidiamond) [inline test/fixtures/v5/fork/sharp_left_name.json]: sharp fork --> sharp left

File: languages/translations/en.json (hunk 1 above)
Comment (allierowan) [reply languages/translations/en.json]: 😂  good catch, LOL

**Comment by allierowan** [comment]
@lyzidiamond thanks for the catch. I fixed that, and also added the new version to the package.json and pushed tags. LMK if it looks good for merge/publish.

**Pull request merged**